src/nostrHelper.ts

```diff
@@ -24,3 +25,49 @@ export const sendPersistEvent = async (kind, tags, content, privkey) => {
 	e.sig = await signEvent(event, privkey);
 	db.events.add(e);
 };
+
+export const getDegreesForPubkey = async (pubkey: string, profiles: IProfile[]) => {
```

Comment:
There is much copy&paste going on here.

Please consolidate. The main diff is that one is starting with follows[0] being all own accounts while the other uses just one account. Please change `nostrHelper.getDegreesForPubkey` to `nostrHelper.getDegreesForPubkeys`, accepting more than one pubkey and make use of it in data.ts